bin/elm-review

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env node
 
 // TODO(@lishaduck): With ESM, this should use TLA.
-require('../lib/main')();
+require('../lib/main').main();
```

Comment:
~~Was just thinking, we should ensure this doesn't break elmls.~~
As far as I can tell, it's just shelling out, correct?
https://github.com/elm-tooling/elm-language-server/blob/d0c9186ec7561abdf7d970be51767304b91d7f5b/src/common/providers/diagnostics/elmReviewDiagnostics.ts#L145-L150